### ISSUE TITLE
[Wallet - Token management] Prepare send modal to display community assets

### ISSUE THREAD

**Issue by noeliaSD** [status-im/status-desktop#12378]
### Description
Depends on issue https://github.com/status-im/status-desktop/issues/12611

[Design](https://www.figma.com/file/eM26pyHZUeAwMLviaS1KJn/%E2%9A%99%EF%B8%8F-Wallet-Settings%3A-Manage-Tokens?type=design&node-id=1%3A16514&mode=design&t=EZE3HMbKPpJc8Wlk-1)

Prepare `sendModal` to be able to display community assets.

![Screenshot 2023-10-07 at 10 09 36](https://github.com/status-im/status-desktop/assets/97019400/737e7583-d3a1-4732-b176-113aff46920b)

Take profit of the task to review if it can be splitted in smaller components (if complex, just a first iteration) and add whatever possible on `storybook`.

**Comment by noeliaSD** [comment]
This task has lower priority than other tasks from the same EPIC. So potential task to be moved to the next iteration.

**Comment by noeliaSD** [comment]
Hey `UI team`. This is a potential task to be picked up without need of `Tokens Management` context.
cc: @caybro, @alexandraB99, @alexjba, @micieslak

**Comment by caybro** [comment]
> Hey `UI team`. This is a potential task to be picked up without need of `Tokens Management` context. cc: @caybro, @alexandraB99, @alexjba, @micieslak

I feel like this better be done by someone from the wallet team?

**Comment by noeliaSD** [comment]
> > Hey `UI team`. This is a potential task to be picked up without need of `Tokens Management` context. cc: @caybro, @alexandraB99, @alexjba, @micieslak
> 
> I feel like this better be done by someone from the wallet team?

Both teams can pick it. That's why I added both labels, whoever is free first! 

One benefit I see if some members of the `UI team` pick it up is that since we have no context on the topic, we can help make the component more isolated from the backend.

**Comment by Khushboo-dev-cpp** [comment]
@benjthayer a question about this design, will the same view be expected to be shown in the Tokens dropdown too?

https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?type=design&node-id=9547-124730&mode=design&t=KrHvD72j2YI9Tc8o-4 ?
